integration-tests/bats/sql-server.bats

```diff
@@ -850,6 +850,51 @@ SQL
     )' ""
 
     server_query repo1 1 "SHOW tables" "" # no tables on master
-    
+
     server_query "repo1/feature-branch" 1 "SHOW Tables" "Table\ntest"
 }
+
+@test "sql-server: SET GLOBAL default branch" {
+    skiponwindows "Has dependencies that are missing on the Jenkins Windows installation."
+
+    cd repo1
+    dolt checkout -b "new"
+    dolt checkout master
+    start_sql_server repo1
+
+    multi_query repo1 1 '
+    select dolt_checkout("new");
+    CREATE TABLE t (a int primary key, b int);
+    INSERT INTO t VALUES (2,2),(3,3);' ""
+
+    server_query repo1 1 "SHOW tables" "" # no tables on master
+
+    server_query repo1 1 "set GLOBAL dolt_default_branch = 'refs/heads/new';" ""
```

Comment:
You should make this also work with just a branch name, not a full ref. Add a test for that